[EN] Split the versions into supported/unsupported

martinemde opened the pull request:
I like the recent changes and wanted to also see the End of Support dates grouped into a different list like the JA version.

<img width="1008" alt="Screenshot 2024-12-03 at 9 59 20 AM" src="https://github.com/user-attachments/assets/f9b45b5c-39f0-49dc-8a34-6e6c00c71c28">

st0012 commented:
<img width="1197" alt="Screenshot 2024-12-01 at 12 07 45" src="https://github.com/user-attachments/assets/c120550e-ba32-4289-a254-358949b86c67">

LGTM

But technically Ruby 3.0 has also passed its EOL (2024-04-23) according to [here](https://www.ruby-lang.org/en/downloads/branches/). Maybe we can reflect that in this change too?

martinemde commented:
Updated. Moved 3.0 under other versions. Changed all rows to be more consistent.

Here's a screenshot:

<img width="808" alt="Screenshot 2024-12-03 at 9 56 08 AM" src="https://github.com/user-attachments/assets/49d711f4-c035-48f5-b7e1-55b4ff88cebb">

st0012 reviewed (approved):
LGTM ❤️
